guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -51,18 +48,11 @@ ifdef::satellite[]
 ----
 endif::[]
 . Install the Pulp Manifest package:
-** On {EL} 8:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # dnf install python39-pulp_manifest
 ----
-** On {EL} 7:
```

Comment:
Keep for Satellite. Also elsewhere

Reply:
Are you sure: this would mean that Satellite keeps packaging pulp_manifest from pulpcore-packaging for EL7.

Reply:
Yes, we need to remove it. I'll create a separate PR for it to be merge before this PR.

Reply:
This has been resolved by rebasing.